detection-rules/link_coinbase_phishing_suspicious_link.yml

```diff
@@ -0,0 +1,30 @@
+name: "Coinbase Phishing Email with Link Analysis"
+description: "This rule detects Coinbase phishing emails with suspicious links"
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and sender.email.domain.domain != "coinbase.com"
+  and (
+    (
+      any(["coinbase", "coin base"], strings.icontains(coalesce(body.html.display_text, body.plain.raw), .))
+      or regex.icontains(coalesce(sender.display_name, subject.subject),
+        '\b[CⒸⓒ𝑪𝒞𝓬𝔠𝕔𝕥𝖈𝖔𝖙𝗌𝗰𝘊𝘤𝙤𝚌𝝰][oₒⓞ𝑜𝒐𝓸𝕠𝖔𝗼𝘐𝙤𝚘𝞸][i𝐼𝑖𝒊𝓲𝕚𝖎𝗜𝘪𝙄𝚒][nⓃ𝑛𝒏𝓷𝔫𝕟𝖓𝙣𝚗]\s?[BⒷⓑ𝑩𝒃𝓫𝔟𝕓𝕥𝗕][aaₐ𝖆𝗮𝘢𝙖𝚊𝛂𝜶𝞪][s𝑆𝒔𝓢𝔰𝕊𝕊𝖘𝖘𝗦𝘴𝙎𝚂][e𝑒𝒆𝓔𝔢𝕖𝕖𝖊𝖊𝗲𝙀𝚎𝝴𝞮]\b')
+    )
+  )
+  and all(body.links,
+    .href_url.domain.domain != "coinbase.com"
+    and (
+      any(beta.binexplode(beta.linkanalysis(.).screenshot),
+        all([
+          "Coinbase",
+          "recovery",
+          "phrase"
+        ], strings.icontains(..scan.ocr.raw, .))
+      )
+      or .href_url.domain.root_domain in $free_subdomain_hosts
```

Comment:
I hit 19/20 known samples, and the 20th I missed, it was because the landing was pulled but it was hosted on $free_subdomain_hosts.... It seems to be a fallback if the page was pulled? Happy to cut it if you think it makes sense.

Reply:
pulling it